dotnet/samples/KernelSyntaxExamples/Example59_OpenAIFunctionCalling.cs

```diff
@@ -90,7 +90,7 @@ private static async Task CompleteChatWithFunctionsAsync(string ask, ChatHistory
         OpenAIFunctionResponse? functionResponse = chatResult.GetOpenAIFunctionResponse();
         if (functionResponse is not null)
         {
-            // If the function returned by OpenAI is an SKFunction registered with the kernel,
+            // If the function returned by OpenAI is an SKFunctionHelper registered with the kernel,
```

Comment:
Is the rename here because if SKFunction becomes an abstract base class it needs to be in abstractions which means it can't reference everything necessary for these FromMethod/FromPrompt factories?

We might want to reconsider how valuable the abstractions / core split actually is.

But if we do still like it and need a different name for this, "Helper" doesn't sound right to me.  I'd prefer "Factory".

Reply:
Agreed naming here needs some thought. If we use `Factory` then we may want to rename the methods.

Reply:
Changed to `KernelFunctionFactory`